modules/platforms/cpp/DEVNOTES.md

```diff
@@ -1,26 +1,27 @@
 ## Build C++
 
 ### Prerequisites
-* C++ compiler supporting C++17;
-* One of build systems: make, ninja, MS Visual Studio, etc;
-* Conan C/C++ package manager 1.X (optional);
-* CMake 3.10+;
-* To build the ODBC driver, it is required to have an ODBC driver manager with headers on your system. On Windows, it 
-  comes with your OS, but on Unix-like systems you may need to install one, for example, unixODBC;
+
+* C++ compiler supporting C++ 17
+* One of the build systems: make, ninja, MS Visual Studio, etc.
+* Conan C/C++ package manager 1.X (optional)
+* CMake 3.10+
+
+To build the ODBC driver, it is required to have an ODBC driver manager with headers on your system. On Windows, it comes with your OS. On Unix-like systems, you may need to install one; for example, unixODBC.
```

Comment:
This file is supposed to be human-readable from IDE so line breaks at column 120 are required.